python-template/{{cookiecutter.placeholder_repo_name}}/.readthedocs.yaml

```diff
@@ -8,8 +8,14 @@ version: 2
 # Build documentation in the docs/ directory with Sphinx
 sphinx:
   configuration: docs/conf.py
+  fail_on_warning: true
```

Comment:
Can you update https://github.com/openedx/edx-cookiecutters/blob/master/CHANGELOG.rst, and especially note this fail on warning change, in addition to the fixes to read the docs builds?